addons/delivery/views/product_packaging_view.xml

```diff
@@ -1,47 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-<record model="ir.ui.view" id="product_packaging_delivery_form">
+<record model="ir.ui.view" id="delivery.product_packaging_delivery_form">
```

Comment:
I bet you have had some issues in a rebase. I don't think you had to modify delivery, am I wrong ?